test/sanity/pep8/README.md

```diff
@@ -24,3 +24,10 @@ Files listed in the [skip list](skip.txt) are not tested by `pep8`.
 ## Removed Files
 
 Files which have been removed from the repository must be removed from the legacy file list and the skip list.
+
+## Running locally
+
+You can do a quick pep8 spotcheck by approximating `command_sanity_pep8` in `test/runner/lib/executor.py` like this:
```

Comment:
No need to reference the internals of `ansible-test` here. A simpler explanation should suffice:

```
The PEP 8 check can be run locally with:
```